### DISCUSSION TITLE
Any plans to use Textual itself to power the Textual website?

### DISCUSSION THREAD

**Discussion by msafi** [Textualize/textual#4273] [General]
I was browsing Textual with my son and he asked if the website is made with Textual, which I thought was an interesting question. 

I don't think the website is made with Textual and was wondering if that can change in the future.

**Comment by davep** [comment]
Textual is a framework for building terminal-based applications, it's not really intended for building websites. That said, using `textual-web`, you *can* embed Textual applications within a website. You'll see this in some places throughout our own site; for example see the [live stopwatch app example on the front page](https://textual.textualize.io/#__tabbed_1_1).

**Comment by willmcgugan** [comment]
It is a little ironic! Textual is great for tools, but regular HTML is still best for serving content.

**Comment by msafi** [comment]
@davep @willmcgugan Thanks!

I worked with TypeScript/JS for years, then recently I started looking for a simple CLI framework for my first Python project. I didn't think something like Textual existed, so I'm very excited to have found it. It seems very comprehensive and well done, with a lot of good ideas from web development. Thanks for making it!
